### ISSUE TITLE
ytep link on main webpage is broken

### ISSUE THREAD

**Issue by zingale** [yt-project/yt#4406]
### Bug report

**Bug summary**

the ytep link 404s

**Code for reproduction**

1. Go to: https://yt-project.org/
2. Hover over "quick links" at the top, and select "YTEPs"

the page is not found.

I don't remember where the webpages live.

```python
# Paste your code here
#
#
```

**Actual outcome**

```
# If applicable, paste the console output here
#
#
```

**Expected outcome**

**Version Information**

  * Operating System:
  * Python Version:
  * yt version:
  * Other Libraries (if applicable):

**Comment by neutrinoceros** [comment]
good catch ! indeed https://ytep.readthedocs.io/en/latest/ is broken, but https://ytep.readthedocs.io/ still works, so I think we should roll with that ?

**Comment by zingale** [comment]
yes, I think that's safest.

**Comment by neutrinoceros** [comment]
the bug is on the website however, not the core yt codebase, so let's migrate the issue
